### DISCUSSION TITLE
How does the user Prompt convert to a Web Search exactly?

### DISCUSSION THREAD

**Discussion by ScottzCodez** [qunash/chatgpt-advanced#182] [Q&A] [answered]
I was wondering what the exact web search (URL) is done with this user Prompt:  (without quotes) 
"do llms trained on github code use the final versions only or do they use the evolution of versions with Pull requests, issues, bug fixes, etc"
  I did a google search with the string and didn't come up with what WebChatGPT said were the Search Results.

I tried to figure out the code, to see how it might modify the prompt before passing it to Google but I only know python.
Thanks.

**Comment by 6DiegoDiego9** [comment, marked as answer]
Unfortunately, WebChatGPT uses Yahoo for web searches, not Google.

It would be great to have it use Google. If cost is the issue, it could use personal API keys: https://developers.google.com/custom-search/v1/introduction

**Reply by ScottzCodez** [reply]
Can't you just google search using the python requests library for free and then parse the results?    
excuse me, I'm ignorant of any issues with that.
